angular/projects/admin/src/app/admin/settings/settings.component.ts

```diff
@@ -84,6 +91,20 @@ export class SettingsComponent implements OnInit, OnDestroy {
     this.domainsFormArray.removeAt(index);
   }
 
+  dialogLanguage() {
+    const dialogRef = this.dialog.open(SettingsDialogManageLanguagesComponent, {
+      data: this.languages
+    });
+    dialogRef.afterClosed().subscribe(result => {
+      if (result !== undefined) {
+        if (result.status === 'submit') {
```

Comment:
You can shorten it to a single line if-statement

```typescript
if (result && result.status === 'submit') {
 // ...
}
```

Reply:
Solved on this commit https://github.com/oddbit/tanam/pull/51/commits/cb5cfa62b2b1b5eb1802ee3baae26563b60ce66b